test/test_main.cpp

```diff
@@ -183,6 +206,14 @@ int main(int argc, char* argv[])
             num_failed = session.run();
         }
 
+        if (num_failed == 0 && cpu) {
+            enable_normalization = true;
```

Comment:
Maintain consistency on how you set the flags.  On the block below it's directly around the `run()` call.  Here, it's around the whole block.  Choose one.

Also applies below.

Recommendation: do it directly around the `run()` call.

Reply:
Fixed.